### ISSUE TITLE
Can't install precompiled version 0.1.19

### ISSUE THREAD

**Issue by RyoWakabayashi** [cocoa-xu/evision#135]
I get an error when trying to install evision on Livebook

```
06:54:50.279 [info] EVISION_PREFER_PRECOMPILED: true; try to download and use the precompiled library.

06:54:50.288 [info] Requested version `0.1.19` has precompiled binaries.

06:54:50.288 [info] Current target `x86_64-linux-gnu` has precompiled binaries.

06:54:50.288 [info] Current NIF version `2.16` has precompiled binaries.

06:54:58.842 [info] Precompiled binary tarball downloaded and saved to /home/livebook/.cache/evision-nif_2.16-x86_64-linux-gnu-0.1.19.tar.gz, sha256=fa65b4889fa490887f8ad180680d8e8f2ebcbab3f7c482a3a42b98cc089204d6

06:54:58.846 [error] Checksum mismatched: downloaded file: /home/livebook/.cache/evision-nif_2.16-x86_64-linux-gnu-0.1.19.tar.gz[sha256=fa65b4889fa490887f8ad180680d8e8f2ebcbab3f7c482a3a42b98cc089204d6], expected:[sha256=b7ef14999221c20d425a46f6d0578ecea3e5b86f120b0d454e6b45bf6778f305]
==> evision
could not compile dependency :evision, "mix compile" failed. Errors may have been logged above. You can recompile this dependency with "mix deps.compile evision", update it with "mix deps.update evision" or clean it with "mix deps.clean evision"
```

No error if version 0.1.18 is specified

**Comment by cocoa-xu** [comment]
Hi @RyoWakabayashi! Sorry, the checksum for `x86_64-linux-gnu` in v0.1.19 was incorrect. I've released v0.1.20 and tested it on a Linux machine. It should work now. :)

**Comment by RyoWakabayashi** [comment]
@cocoa-xu 

Thank you.
I confirm that v0.1.20 can be installed.
